### PR TITLE
Pre post

### PR DESCRIPTION
**Pull request by taoleicn**
Option for post layer norm.

### CODE REVIEW

File: sru/modules.py [hunk 1]
```diff
@@ -427,7 +438,8 @@ def __init__(self,
                  nn_rnn_compatible_return: bool = False,
                  custom_m: Optional[Union[nn.Module, List[nn.Module]]] = None,
                  proj_input_to_hidden_first: bool = False,
-                 amp_recurrence_fp16: bool = False):
+                 amp_recurrence_fp16: bool = False,
+                 post_ln: bool = False):
```
Comment (taoleicn) [inline sru/modules.py:442]: @hpasapp can we change the name from `post_ln` to sth like `post_normalization` or `normalize_after`?
e.g.
https://github.com/pytorch/fairseq/blob/1bc83c703ad70d7f62c1e54b197e29b95d07b1f0/fairseq/modules/transformer_layer.py#L130

Comment (hpasapp) [reply sru/modules.py]: addressed in b8235f0

**Comment by taoleicn** [comment]
@hpasapp one last thing: can we add `normalize_after` to the torchscript unit test?
:P

**Comment by hpasapp** [comment]
> @hpasapp one last thing: can we add `normalize_after` to the torchscript unit test?
> :P

addressed in 265f75d

**Comment by taoleicn** [comment]
Amazing @hpasapp :) 🎉
